test/server/action_creators_spec.js

```diff
@@ -134,4 +146,51 @@ describe('server action creators', () => {
       done();
     });
   });
+
+  it('changeEntryStatus', (done) => {
+    let dispatched;
+    const orders = [
+      {
+        id: 1,
+        status: constants.OPEN,
+        entries: [
+          {
+            id: 2,
+            status: constants.OPEN,
+            name: '',
+            comment: ''
+          }
+        ]
+      }
+    ];
+    function dispatch(action) {
+      dispatched = action;
+    }
+    function getState() {
+      return { orders };
+    }
+    const expected = {
+      type: constants.SET_STATE,
+      state: {
+        orders: [
+          {
+            id: 1,
+            status: constants.OPEN,
+            entries: [
+              {
+                status: constants.DELIVERED,
+                name: '',
+                comment: ''
+              }
+            ]
+          }
+        ]
+      }
+    };
+
```

Comment:
This comment is for generating test data in general - I've been debating on creating a generator, like generator.order({entries, status, id}).build() to remove the boilerplate code in our tests. It would have defaults, and you would explicitly set the attributes you are testing for. This way our tests would be more readable. Let me know what you think. If you like it, we can create a task for it.